gordon/schema/definitions/google-audit-log.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-03/schema",
+
+  "title": "Google Audit Log",
```

Comment:
This is quite Google-specific. @steven and I had a chat on Monday, and I am going to write something up (like a API design doc). 

Basically, everything about Gordon core will be pretty abstract, and extensions/plugins would be external packages/python dependencies that are brought into Gordon core via configuration. For example, if you want to consume from Google PubSub and/or use Google Cloud DNS and/or use Google Compute Engine for the source of truth, there'd be a separate package, something like `pip install gordon-dns-gcp`. 

So I'm thinking that we should define a basic schema(s) - sort of like an AbstractBaseClass in Python - so plugins like gcp can implement a more specific schema(s).

@steven any thoughts?

Reply:
Yeah, unfortunately since this one is based off of the log messages Google creates, it is quite tied to Google. I'll hold off making any changes here, until we've had time to change the packaging scheme a bit.